service_capacity_modeling/models/org/netflix/kafka.py

```diff
@@ -207,7 +210,8 @@ def _estimate_kafka_cluster_zonal(
         cluster_size=lambda x: next_n(x, zones_per_region),
         min_count=max(min_count, 1),
         # Sidecars and Variable OS Memory
-        reserve_memory=lambda x: base_mem,
+        # Kafka currently uses 8GiB fixed, might want to change to min(30, x // 2)
```

Comment:
Thinking more about this, we do not want to allocate a lot of RAM to JVM heap for Kafka as we want to leave most memory for the page cache so consumers don't have to do disk reads. The JVM heap usage is low in kafka brokers even at high scale due to zero-copy transfers

Reply:
The heap is probably pretty useful for Kafka, but agreed if y'all end up changing your heap formula we should refactor this.